album_search.py

```diff
@@ -0,0 +1,43 @@
+import thefuzz as thefuzz
```

Comment:
The `as` keyword is only necessary if you want to reference an import by another name. To make this a little cleaner, change it to the following:

```python
import thefuzz
```

That said, I'll admit that that I think the way it's written right now looks incredibly funny.

Reply:
I was contemplating just importing it as tf but then I was like, thefuzz looks funnier lol and never changed it

Reply:
Yup! I understand. You can leave it as `thefuzz` if you want. 😆 Just get rid of the unnecessary `as` clause.